src/test-reader/test-parser.js

```diff
@@ -28,22 +28,25 @@ class TestParser extends EventEmitter {
             system: { ctx, mochaOpts },
         } = config;
 
-        global.hermione = {
+        const toolGlobals = {
             browser: browserVersionController.mkProvider(config.getBrowserIds(), eventBus),
             config: ConfigController.create(eventBus),
             ctx: _.clone(ctx),
             only: OnlyController.create(eventBus),
             skip: SkipController.create(eventBus),
         };
 
+        global.testplane = toolGlobals;
+        global.hermione = toolGlobals;
```

Comment:
all public api instances of `hermione*` are also exported as copies (with Object.is equality) for `testplane*`